deploy-manage/deploy/cloud-on-k8s/install-using-helm-chart.md

```diff
@@ -17,16 +20,14 @@ The minimum supported version of Helm is 3.2.0.
 ::::
```

Comment:
consider putting the helm minimum version in a sub variable so it's consistent between here and https://docs-v3-preview.elastic.dev/elastic/docs-content/pull/380/deploy-manage/deploy/cloud-on-k8s#k8s-supported

Reply:
Added to the project issue as a new task for a later stage. I prefer to change the variables file in a new PR after this is merged.